anabatic/src/AutoSegment.cpp

```diff
@@ -2822,7 +2829,7 @@ namespace Anabatic {
     DbU::Unit    verticalWidth   = Session::getDVerticalWidth();
     if (not Session::getAnabatic()->getConfiguration()->isGMetal(hurricaneSegment->getLayer())) {
       size_t depth = Session::getAnabatic()->getConfiguration()->getLayerDepth( hurricaneSegment->getLayer() );
-      if (depth > 2) {
+      if ((depth == 0) or (depth > 2)) {
```

Comment:
It's unclear to me what's going on here

Reply:
Mmm. It means that if either we are at depth 0 (``M1``) or *above* depth 3 (``M3``), we use the specific wire width and perpandicular pitch of those layers. Otherwise (``M2`` or ``M3``) we use the *default* defined wire width and perpandicular. Re-reading that code make me thinks I could do it in a more clear way.

Reply:
Maybe at least commenting it, as it's difficult to understand for someone that doesn't know the codebase

Reply:
Yes. But, this is typically something I should rewrite and cleanup.